docs/content/engines/overview.md

```diff
@@ -31,7 +31,7 @@ under the License.
 |                                     Engine                                      |    Version    |  Batch Read | Batch Write |  Create Table |  Alter Table  | Streaming Write  |  Streaming Read  | Batch Overwrite |  DELETE & UPDATE   | MERGE INTO  | Time Travel |
 |:-------------------------------------------------------------------------------:|:-------------:|:-----------:|:-----------:|:-------------:|:-------------:|:----------------:|:----------------:|:---------------:|:------------------:|:-----------:|:-----------:|
 |                                      Flink                                      |  1.15 - 1.20  |     ✅      |      ✅      |      ✅       |  ✅(1.17+)   |        ✅        |       ✅        |        ✅        |    ✅(1.17+)      |      ❌     |      ✅      |
-|                                      Spark                                      |   3.1 - 3.5   |     ✅      |   ✅(3.2+)   |      ✅       |      ✅      |      ✅(3.3+)    |    ✅(3.3+)     |     ✅(3.2+)     |     ✅(3.2+)      |   ✅(3.2+)  |   ✅(3.3+)   |
+|                                      Spark                                      |   3.2 - 3.5   |     ✅      |   ✅(3.2+)   |      ✅       |      ✅      |      ✅(3.3+)    |    ✅(3.3+)     |     ✅(3.2+)     |     ✅(3.2+)      |   ✅(3.2+)  |   ✅(3.3+)   |
```

Comment:
Not only removing 3.1, but also removing `3.2+`, `3.2+` is meaningless now.

Reply:
OK